tests/python-e2e/ops.py

```diff
@@ -18,6 +18,49 @@ def test_read_write_family(self):
             self.assertEqual(read_text, TEXT)
             rw_file.close()
 
+    def test_fopen(self):
+        """
+        Opens a file in "/tmp/"
+        """
+        file_path, _ = self._create_new_tmp_file()
+        self.assertFalse(self._check_path_exists_on_host(file_path))
+        with open(file_path, "r") as f:
+            f.seek(0)
+            txt = f.readline()
```

Comment:
Does it call fopen? If so, aren't we already performing a similar test in a previous function?